src/migrations/migrations/28.0.0snap1/1_add-maintenance-window-start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh -e
+
+# Starting from Version 28 Nextcloud recommends to set a default time window for resource heavy background jobs.
+# See https://docs.nextcloud.com/server/28/admin_manual/configuration_server/background_jobs_configuration.html#maintenance-window-start
+
+# Path to config.php file
+config_file="$SNAP_DATA/nextcloud/config/config.php"
+
+# Searching text (config name)
+search_text="maintenance_window_start"
+
+# New config text
+insert_text="'maintenance_window_start' => 1,"
+
+# Check if config name exists
+if grep -q "$search_text" "$config_file"; then
+    echo "The config '$search_text' is already specified in '$config_file'."
+else
+    # Paste the the default config value in the last line
+    sed -i '$i\'"$insert_text" "$config_file"
```

Comment:
Can you explain the logic behind this command @Pilzinsel64 ? My bash foo is a little bit rusty 😓

Reply:
...and mine non existent. Wrote that too lang ago. 😄
As far as I remember does `sed` repalce a text in-place (parameter `-i`) in the file `"$config_file"`. `$i` means to insert the new text (``) at before the last line. While `\` should somewhow escape the `"`. I'm not sure how to escape it else plus keeping the variable.